winit-x11/src/window.rs

```diff
@@ -1238,10 +1237,14 @@ impl UnownedWindow {
 
             let old_surface_size = PhysicalSize::new(width, height);
             let surface_size = Arc::new(Mutex::new(PhysicalSize::new(new_width, new_height)));
-            app.window_event(event_loop, self.id(), WindowEvent::ScaleFactorChanged {
-                scale_factor: new_monitor.scale_factor,
-                surface_size_writer: SurfaceSizeWriter::new(Arc::downgrade(&surface_size)),
-            });
+            app.window_event(
+                event_loop,
+                self.id(),
+                WindowEvent::ScaleFactorChanged {
+                    scale_factor: new_monitor.scale_factor,
+                    surface_size_writer: SurfaceSizeWriter::new(Arc::downgrade(&surface_size)),
+                },
```

Comment:
This is just a reformatting diff, right?  was there anything you needed to change here?